core/src/main/java/io/kestra/core/models/flows/State.java

```diff
@@ -172,7 +172,8 @@ public enum Type {
         KILLED,
         CANCELLED,
         QUEUED,
-        RETRYING;
+        RETRYING,
+        RETRIED;
 
```

Comment:
I don't understand why the RETRIED state is needed, it seems the same as RETRYING.
We already have a lot of different state and I fear our user will be a bit overwhelmed with all those different states, moreover if those are very close to each other.

Reply:
I agree that we're getting a lot of status, but those are differents like for KILLED and KILLING
RETRIED mean the execution isn't running anymore, but a new will start soon (based on the interval set)
While RETRYING mean the execution is still running and will start a new execution

I can remove the RETRYING and keep only RETRIED, but only RETRYING mean that the execution is still doing something